src/components/guess.js

```diff
@@ -27,6 +40,79 @@ class Guess extends Component {
       return null;
     }
   }
+
+  stintToString(stint) {
+    let stint_info = "";
+
+    if (!stint) {
+      return "";
+    }
+
+    let type = stint.get('stint_type')
+    if (!type) {
+      return "";
+    }
+
+    if (type === 'retreat') {
+      return stint.get('short_name');
+    }
+
+    switch (type) {
```

Comment:
Now that the `'retreat'` case is a simple `return`, I think it makes sense to merge the `if` statement into the `switch` statement, like:

```javascript
switch (type) {
  case 'retreat':
    return stint.get('short_name');
  case 'residency':
```